### PR TITLE
feat: Rudimentary skip button

### PR DESCRIPTION
**Pull request by ljkbennett**
Gives everyone the right to unilaterally skip :( Just an interim until we put in
"vote to skip"

### CODE REVIEW

File: apps/juicebox_web/assets/js/videos/video.scss [hunk 1]
```diff
@@ -33,9 +33,20 @@
...
   display: flex;
   width: 100%;
-  height: 50px;
-  padding: 0 50px;
+  min-height: 50px;
+  padding: 0 50px 0 0;
   font-size: 17px;
   background-color: #f2f2f2;
   align-items: center;
+  h3 {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/videos/video.scss:41]: Rule declaration should be preceded by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)

File: apps/juicebox_web/assets/js/videos/video.js [hunk 2]
```diff
@@ -1,10 +1,17 @@
 import React, { Component, PropTypes } from 'react';
 import classnames from 'classnames';
+import show from '../lib/show-when';
 import styles from './video.scss';
 import ExpandButton from './buttons/expand.svg';
 import MinimiseButton from './buttons/minimise.svg';
+import Boo from './buttons/boo.svg';
 
 class Video extends Component {
+  skipPlaying = () => {
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/videos/video.js:10]: Parsing error: Unexpected token =

File: apps/juicebox_web/assets/js/videos/actions.js [hunk 3]
```diff
@@ -29,12 +30,27 @@ function toggleExpanded() {
   }
 }
 
+function skipPlaying({ streamId }) {
+  console.log(streamId);
```
Comment (ljkbennett) [inline apps/juicebox_web/assets/js/videos/actions.js]: Unexpected console statement.

[Read more about it here.](http://eslint.org/docs/rules/no-console)

**Comment by littleowllabs** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 3 possible new issues (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/81.

**Comment by ljkbennett** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 2 possible new issues (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/81.

File: apps/juicebox_web/assets/js/videos/video.js [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
+    console.log("props", this.props);
```
Comment (OCannings) [inline apps/juicebox_web/assets/js/videos/video.js]: console.log

**Review by OCannings — CHANGES_REQUESTED** [review]

**Review by OCannings — APPROVED** [review]

**Pull request merged**